From: =?UTF-8?q?Carl=20F=C3=BCrstenberg?= <carl@excito.com>
Date: Fri, 21 Oct 2011 15:33:41 +0200
Subject: Enable kernel options to make iotop work

Enable options CONFIG_TASKSTATS, CONFIG_TASK_DELAY_ACCT, and
CONFIG_TASK_IO_ACCOUNTING
---
 arch/arm/configs/bubba3_defconfig |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/bubba3_defconfig b/arch/arm/configs/bubba3_defconfig
index 141f381..a85a07b 100644
--- a/arch/arm/configs/bubba3_defconfig
+++ b/arch/arm/configs/bubba3_defconfig
@@ -1,6 +1,10 @@
 CONFIG_EXPERIMENTAL=y
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
+CONFIG_TASKSTATS=y
+CONFIG_TASK_DELAY_ACCT=y
+CONFIG_TASK_XACCT=y
+CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_IKCONFIG=y
 CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=15
@@ -239,6 +243,7 @@ CONFIG_USB_ZD1201=m
 CONFIG_USB_NET_RNDIS_WLAN=m
 CONFIG_RTL8187=m
 CONFIG_ATH_COMMON=m
+CONFIG_ATH_USER_REGD=y
 CONFIG_ATH5K=m
 CONFIG_ATH9K=m
 CONFIG_ATH9K_HTC=m
-- 
